Tuist/ProjectDescriptionHelpers/Target+Initializing.swift

```diff
@@ -0,0 +1,46 @@
+import ProjectDescription
+
+extension Target {
+
+    public static func mainTarget(name: String) -> Target {
+        return Target(
+            name: name,
+            platform: .iOS,
+            product: .app,
+            bundleId: "co.nimblehq.\(name)",
+            infoPlist: .extendingDefault(with: [
+                "UILaunchScreen": [:]
+            ]),
+            sources: ["\(name)/Sources/**"],
+            resources: ["\(name)/Resources/**"]
+        )
+    }
+    
+    public static func testsTarget(name: String) -> Target {
+        return Target(
+            name: name + "Tests",
```

Comment:
```suggestion
            name: "\(name)Tests",
```
To keep the consistency